src/app/app-utils.ts

```diff
@@ -68,6 +68,11 @@ export class AppUtils {
    * Set the active property on the navigation items.
    */
   public static setActiveLink(items: NavigationItem[], currentUrl: string): NavigationItem[] {
+
+    if (currentUrl.indexOf('booking') > 0 ) {
```

Comment:
Will it not show on all URLs having booking?

Reply:
Modified the logic and Added additional check whether the user isBookableAndJudicialRole